presto-native-execution/presto_cpp/presto_protocol/Connectors.cpp

```diff
@@ -48,6 +48,12 @@ void registerTpchConnector() {
   registerConnector("tpch", "tpch");
 }
 
+void registerSystemConnector() {
+  registerConnector("$system", "$system");
```

Comment:
Is there any significance to each of the catalog names?
How do `$system`, `system`, and `$system@system` differ?

Reply:
So while the connector is named @system@system, the protocol objects have @system, system and $system@system usages of connector. I don't know the origin of this and its hard to trace from the code.

@tdcmeehan : Do you have any insights here ?